packages/core/src/routes/interaction/utils/verify-user.ts

```diff
@@ -0,0 +1,29 @@
+import RequestError from '#src/errors/RequestError/index.js';
+import { verifyUserPassword } from '#src/lib/user.js';
+import assertThat from '#src/utils/assert-that.js';
+
+import type { PasswordIdentifierPayload, UserIdentity } from '../types/guard.js';
+import findUserByIdentity from './find-user-by-identity.js';
+
+export const verifyUserByVerifiedPasscodeIdentity = async (identifier: UserIdentity) => {
+  const user = await findUserByIdentity(identifier);
```

Comment:
can we use the name `findUserByIdentifier`? Because we have `user.identities`

Reply:
get it will refactor in the following PR.